about.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Web Development Course Summit | About</title>
+  </head>
+  <body>
+    <!-- header -->
+    <header>
+      <!--mobile menu-->
+      <div class="mobile-menu show-menu">
+        <img id="close-hamburger" src="./assets/img/close-btn.png" />
+        <ul class="flex-column">
+          <li class="menu-item">
+            <a class="text-20 custom-white" href="index.html">Home</a>
```

Comment:
it works on my end.

Reply:
Except you mean "home" on the navigation menu and if that's the case. The design didn't have it because the logo takes you to the home from anywhere and that was also stated in the requirement. Please let me know if I got what you're talking about right or not. Screenshots will help too.